src/mudlet-mapper.xml

```diff
@@ -3493,11 +3493,13 @@ mmp.echo("We're connected to Lithmeria.")</script>
 						<string>There is a door in the way, to the</string>
 						<string>There is a door in the way.</string>
 						<string>A closed door is in the way. You need to OPEN DOOR</string>
+						<string>There is an? (walnut|pine|oak|iron|reinforced) door in the way.</string>
```

Comment:
Is "`There is an? (walnut|pine|oak|iron|reinforced) door in the way.`" the correct string here? It looks wrong, but then I don't play IRE MUDs so I do not know for sure.

Reply:
This script is from external repository that we pull in as-is, did you want to give your feedback on https://github.com/IRE-Mudlet-Mapping/ire-mapping-script